packages/server/app/Controllers/Http/SearchUsersController.ts

```diff
@@ -95,35 +95,38 @@ export default class SearchUsersController {
             requestingUser.following.length > 0;
 
         const relateduser = await User.findOrFail(userID);
+        //Note: cannot load relationship after the loadAggregate block
         await relateduser.load('playlistsVisibilitySetting');
         await relateduser.load('relationsVisibilitySetting');
-        await relateduser.load('mpeRooms');
-        await relateduser.load('following');
-        await relateduser.load('followers');
 
         const {
             nickname: userNickname,
             playlistsVisibilitySetting,
             relationsVisibilitySetting,
         } = relateduser;
 
+        await relateduser
+            .loadCount('following')
+            .loadCount('followers')
+            .loadCount('mpeRooms');
+
         const playlistsCounter =
             getUserProfileInformationDependingOnItsVisibility({
-                fieldValue: relateduser.mpeRooms.length,
+                fieldValue: parseInt(relateduser.$extras.mpeRooms_count),
```

Comment:
It seems that [the radix is not necessarily 10](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseInt#parameters) and needs to be provided to `parseInt` explicitly.
Usually I use `Number()` instead of `parseInt()` because `parseInt()` [parses the input](https://thisthat.dev/number-constructor-vs-parse-int/), while `Number()` coerces its type.

Reply:
updating